src/common/deleteItem.js

```diff
@@ -0,0 +1,10 @@
+const deleteItem = (e) => {
+  e.stopPropagation();
+  const index = e.currentTarget.parentNode.childNodes[0].innerHTML;
+  const storedList = JSON.parse(localStorage.getItem('toDolist'));
+  storedList.splice(index, 1);
+  localStorage.setItem('toDolist', JSON.stringify(storedList));
+  window.location.reload();
+};
+/* eslint-disable */
+export { deleteItem }
```

Comment:
- [ ] As I mentioned in the previous comment(s), Kindly make sure you fix all your `linters` errors instead of disabling them as you did for best practices and clean code.

Reply:
Changed the same.